build.rs

```diff
@@ -147,10 +151,10 @@ fn main() {
                 println!("cargo:rustc-env=PTXED={}/bin/ptxed", val);
             } else {
                 build_libipt();
-                c_build.include(&format!("{}/inst/include/", C_DEPS_PATH));
-                c_build.flag(&format!("-L{}/inst/lib", C_DEPS_PATH));
-                println!("cargo:rustc-link-search={}/inst/lib", C_DEPS_PATH);
-                println!("cargo:rustc-env=PTXED={}/inst/bin/ptxed", C_DEPS_PATH);
+                c_build.include(&format!("{}/inst/include/", c_deps_path_abs.display()));
```

Comment:
This feels like we want to call `display` once on `c_deps_paths_abs`.

Reply:
Could do yeah.